test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -761,12 +759,16 @@ def __evaluate_core_file(self, file_path, core_dir):
         :param core_dir: path to directory to store core information
         :type core_dir: str
 
+
         :returns: True if this was a valid core file, otherwise False
         """
         if not os.path.isfile(file_path):
             return False
 
-        file_header = magic.from_file(file_path)
+        # Get the header of this file
+        ret = self.du.run_cmd(cmd=["file", file_path], sudo=self.sudo)
+        assert(ret['err'] is None or len(ret['err']) == 0)
```

Comment:
Codacy is complaining that this assert call will be optimized out in the *.pyo file. If the behavior is critical to the test, then you should raise an error here or fail the test. The rest looks fine.

Reply:
I read that, and the reason that it mentions doesn't apply here. So, I'm going to leave it as is. Thanks for the sign-off!

Reply:
I'm with @mike0042 here.  You put in the assert for a reason and it will be optimized out.  If you click the arrow next to it, it has example code of the correct way to handle the situation.  It also says using an assert() is not considered good style.

Reply:
It says that it gets optimized out when used in a particular way, which I'm not doing here, but if it's not considered good style then Ill change it.